index.js

```diff
@@ -1,188 +1,152 @@
+                        default: '1'
+                    }
+				],
+				callback: async (event) => {
+					const id = await this.parseVariablesInString(event.options.output)
+
+					this.setOutput(id, event.options.para)
+				},
+			},
+			
+		})
 	}
+    updateFeedbacks() {
+        this.setFeedbackDefinitions({
+            outputs: {
+                type: 'boolean',
+                name: 'Outputs',
+                description: 'Get the status of the outputs',
+                options: [
+                    {
+                        type:'number',
+                        label: 'Output',
+                        id: 'output',
+                        default: '1',
+
+                    }
+                ],
+                defaultStyle: {
+                    bgcolor: combineRgb(255,255,0),
+                },
+                callback: (feedback,) => {  
+                    if (this.outputs.length > 0) {
+                        let output = this.outputs.find(element => element.ID.toString() == feedback.options.output)
+                        if(output){
+                        return output.State == 1
+                        } else {
+                            return false
+                        }
+                    }
+                    return false
+                }}});
+
+    }
+    async updatePresets() {
+        let presets = [];
+        let url = `http://${this.config.ip}/netio.json`;
+        let response = await axios.get(url).catch(error => {
+            this.log('error', error.message)
+        } );
+        this.outputs = response.data.Outputs;
+        this.outputs.forEach(element => {
+            presets.push({
+                type: 'button',
+                category: 'Outputs',
+                name: 'Output ' + element.ID,
+                style:{
+                    text:"Output "+element.ID+" ",
+                    color:combineRgb(255,255,255),
+                    size:'auto',
+                    bgcolor:combineRgb(0,0,0),
+                },
+                steps:[{down:[{actionId:'setOutput', options:{output:element.ID, para:1}}]},{down:[{actionId:'setOutput', options:{output:element.ID, para:0}}]}],
+                feedbacks: [{feedbackId:'outputs', options:{output:element.ID},style:{
+                    bgcolor: combineRgb(255,255,0),
+                    color: combineRgb(0,0,0),
+                }}]
+
+        })});
+        this.setPresetDefinitions(presets);
+    }
+    async getStatus() {
+        let url = `http://${this.config.ip}/netio.json`;
+        let response = await axios.get(url).catch(error => {
+            this.log('error', error.message)} );
+        this.outputs = response.data.Outputs;
+        this.checkFeedbacks('outputs'); 
+    }   
+    async setOutput(output, para) {
+        let url = `http://${this.config.ip}/netio.json`;
+        let body = JSON.parse(`{ "Outputs":[ { "ID":${output}, "Action":${para} } ] }`);
```

Comment:
with output and para being variables, you may want to put a try/catch around the `JSON.parse` in case it fails